_layouts/home.html

```diff
@@ -32,7 +32,7 @@
 
         {% include google-tag-manager.html %}
 
-        <a href="https://github.com/gocd/gocd/">
+        <a href="https://github.com/gocd/gocd/" analytics-label="github-fork-me">
```

Comment:
> I presume that the link will be prevented from being traversed to, while the analytics information is sent

That's not right. Tracking events sent asynchronizely in a separate event handler,  and will not block any other actions. So in the case clicking a link result in load another page, there is a chance event sending is not fast enough to happen, but there is no chance it breaks clicking action. 

In the case event sending not happen because of new page loading, the event is stored in the localStorage (fall back to cookies if localStorage not available),  and will be sent along with following events happened. And event storing logic on server side is robust enough to get the order right.

So if Journey is down, you may lost some events, but user should not be bothered.

The tracking code is right here, you can take a look. https://github.com/journey-app/journey-tracker.js